images/nginx-slim/Makefile

```diff
@@ -1,14 +1,44 @@
+# Copyright 2017 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 all: push
 
 # 0.0.0 shouldn't clobber any released builds
-TAG = 0.15
-PREFIX = gcr.io/google_containers/nginx-slim
+TAG = 0.16
+REGISTRY = gcr.io/google_containers
+ARCH ?= $(shell go env GOARCH)
+
+IMGNAME = nginx-slim
+IMAGE = $(REGISTRY)/${IMGNAME}-$(ARCH)
```

Comment:
Both brace and parentheses are same per https://www.gnu.org/software/make/manual/html_node/Reference.html#Reference, but lemme maintain the consistency by using parentheses